core/example/payment-flow-merchant/main.ts

```diff
@@ -105,14 +105,7 @@ async function main() {
     console.log('\n6. 🔗 Validate transaction \n');
 
     try {
-        await validateTransactionSignature(
-            connection,
-            signature,
-            MERCHANT_WALLET,
-            amount,
-            undefined,
-            reference
-        );
+        await validateTransactionSignature(connection, signature, MERCHANT_WALLET, amount, undefined, reference);
```

Comment:
I didn't touch this file, I think my prettier caught it but going to leave in this PR